Gruntfile.js

```diff
@@ -186,6 +187,43 @@ module.exports = function( grunt ) {
 		}, //makepot
 	} );
 
+	/**
+	 * Helper task to keep all the scripts from WordPress core that are required by the Jasmine tests up to date.
+	 *
+	 * Note the list of scripts needs to be kept up to date.
+	 *
+	 * Pass the location of your WordPress installation using --abspath.
+	 */
+	grunt.registerTask( 'updateJasmineCoreScripts', function() {
+
+		var abspath = grunt.option( "abspath" );
+
+		if ( ! grunt.file.exists( abspath + '/wp-includes' ) ) {
+			grunt.fail.fatal( 'WordPress test install not found. See readme for more information. Currently looking here: ' + abspath );
+		}
+
+		var scripts = [
```

Comment:
Could we define this variable further up, and use it in our vendor definition too?